test/integration-test-main.sh

```diff
@@ -567,16 +588,37 @@ function test_update_metadata_external_large_object() {
     aws_cli s3 cp "${TEMP_DIR}/${BIG_FILE}" "s3://${TEST_BUCKET_1}/${OBJECT_NAME}" --no-progress
     set_xattr key value "${TEST_SETXATTR_FILE}"
     cmp "${TEST_SETXATTR_FILE}" "${TEMP_DIR}/${BIG_FILE}"
+    XATTR_VALUE=$(get_xattr key "${TEST_SETXATTR_FILE}")
+    if [ -z "${XATTR_VALUE}" ] || [ "${XATTR_VALUE}" != "value" ]; then
+        echo "could not read xattr(key) value."
+        return 1
+    fi
 
     #
     # remove xattr
     #
     # "%7B%22key%22%3A%22dmFsdWU%3D%22%7D" = {"key":"value"}
     #
-    OBJECT_NAME=$(basename "${PWD}")/"${TEST_RMXATTR_FILE}"
-    aws_cli s3 cp "${TEMP_DIR}/${BIG_FILE}" "s3://${TEST_BUCKET_1}/${OBJECT_NAME}" --no-progress --metadata xattr=%7B%22key%22%3A%22dmFsdWU%3D%22%7D
-    del_xattr key "${TEST_RMXATTR_FILE}"
-    cmp "${TEST_RMXATTR_FILE}" "${TEMP_DIR}/${BIG_FILE}"
+    # [FIXME]
+    # For macos, the xattrs value specified with the "--metadata" option cannot
```

Comment:
This seems strange since this should just be an HTTP header value.  However let's merge this PR now and follow up later, perhaps after the curl AWS client PR merges.